src/pages/core/guides/javascript/how-to-create-a-core-collection.md

```diff
@@ -0,0 +1,414 @@
+  image: imageUri[0],
+  external_url: 'https://example.com',
+  attributes: [
+    {
+      trait_type: 'trait1',
+      value: 'value1',
+    },
+    {
+      trait_type: 'trait2',
+      value: 'value2',
+    },
+  ],
+  properties: {
+    files: [
+      {
+        uri: imageUri[0],
+        type: 'image/jpeg',
+      },
+    ],
+    category: 'image',
+  },
+}
+```
+
+The fields here include:
+
+| field         | description                                                                                                                                                                               |
+| ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| name          | The name of your Collection.                                                                                                                                                                     |
+| description   | The description of your Collection.                                                                                                                                                              |
+| image         | This will be set to the `imageUri` (or any online location of the image) that we uploaded previously.                                                                                     |
+| animation_url | This will be set to the `animation_ulr` (or any online location of the video/glb) that you've uploaded.                                                                                   |
+| external_url  | This would link to an external address of your choice. This is normally the projects website.                                                                                             |
+| attributes    | Using an object og `{trait_type: vlue, "value": "value1"}`                                                                                                                                |
+| image         | This will be set to the `imageUri` (or any online location of the image) that we uploaded previously.                                                                                     |
+| properties    | Contains the `files` field that takes an `[] array` of `{uri: string, type: mimeType}`. Also contains the category field which can be set to `image`, `audio`, `video`, `vfx`, and `html` |
+
+After creating the metadata, we need to Upload that as a Json file, so we can get an Uri to attach to our Collection
+
+```js
+// Call upon umi's `uploadJson()` function to upload our metadata to Arweave via Irys.
+const metadataUri = await umi.uploader.uploadJson(metadata).catch((err) => {
+  throw new Error(err)
+})
+```
+
+Now we should finally have the URI of JSON file stored in the `metadataUri` providing it did not throw any errors.
+
+### Minting the Core Collection
```

Comment:
in the anchor example you show how to use plugins. Maybe we should show it in JS, too?